src/client/mod.rs

```diff
@@ -77,6 +76,8 @@ pub struct ClientBuilder<'a> {
     intents: GatewayIntents,
     #[cfg(feature = "cache")]
     timeout: Option<Duration>,
```

Comment:
Since you have removed the above feature gate of the import, you can also remove the one here. It's not necessary anymore.
```suggestion
    timeout: Option<Duration>,
```